crates/trbfv/src/helpers.rs

```diff
@@ -6,21 +6,49 @@
 
 use crate::shares::ShamirShare;
 use anyhow::Result;
+use e3_bfv_helpers::BfvParamSets;
 use e3_crypto::{Cipher, SensitiveBytes};
 use fhe::mbfv::PublicKeyShare;
 use fhe::{
-    bfv::{self, BfvParameters},
+    bfv::{self, BfvParameters, SecretKey},
     trbfv::{SmudgingBoundCalculator, SmudgingBoundCalculatorConfig},
 };
 use fhe_math::rq::Poly;
 use fhe_traits::{DeserializeWithContext, Serialize};
 use num_bigint::BigUint;
 use petname::Petnames;
+use serde::{Deserialize as SerdeDeserialize, Serialize as SerdeSerialize};
 use std::{
     hash::{DefaultHasher, Hash, Hasher},
     sync::Arc,
 };
 
+/// Simple serialization wrapper for SecretKey coefficients.
+#[derive(SerdeSerialize, SerdeDeserialize)]
+struct SecretKeyData {
+    coeffs: Box<[i64]>,
+}
+
+/// Serialize a BFV SecretKey to bytes.
+pub fn serialize_secret_key(sk: &SecretKey) -> Result<Vec<u8>> {
+    let data = SecretKeyData {
+        coeffs: sk.coeffs.clone(),
+    };
+    Ok(bincode::serialize(&data)?)
+}
+
+/// Deserialize a BFV SecretKey from bytes.
+pub fn deserialize_secret_key(bytes: &[u8], params: &Arc<BfvParameters>) -> Result<SecretKey> {
+    let data: SecretKeyData = bincode::deserialize(bytes)?;
+    Ok(SecretKey::new(data.coeffs.to_vec(), params))
+}
+
+/// TODO: Make this modular
```

Comment:
If this is node wide then it might be worth making it hidden configuration at the app_config level which means it needs to be injected instead of pulled from a function like this. 

So my gut instinct is that we should probably inject or hardcode this as the ThresholdKeyshare actor is being initialized and then use the self value.

If this is dynamic based on E3 then that is different again but I don't think that is the case. I would like to have a conversation with @hmzakhalid and @0xjei on this as well some of the other parameter configuration issues.